frontend/components/TableContainer/DataTable/DataTable.tsx

```diff
@@ -34,6 +35,7 @@ const DataTable = ({
   columns: tableColumns,
   data: tableData,
   isLoading,
+  sortByAPI = false,
```

Comment:
I wonder if this would be more clear to call `manualSortBy` so that we are consistent with the `useTable` API? I don't feel strongly.